### PR TITLE
drafter 3.2.7 (new formula)

### PR DESCRIPTION
**Pull request by vassilevsky**

### CODE REVIEW

File: Formula/drafter.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class Drafter < Formula
+  desc "API Blueprint document parser"
+  homepage "https://apiblueprint.org/"
+  url "https://github.com/apiaryio/drafter/releases/download/v3.2.7/drafter-v3.2.7.tar.gz"
+  sha256 "a2b7061e2524804f153ac2e80f6367ae65dfcd367f4ee406eddecc6303f7f7ef"
+
+  def install
+    system "./configure", "--shared"
+    system "make", "drafter"
+    bin.install "bin/drafter"
```
Comment (ilovezfs) [inline Formula/drafter.rb]: Is there a `make install`?

File: Formula/drafter.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    (include + "drafter").install "src/drafter.h"
+    lib.install "build/out/Release/libdrafter.dylib"
+  end
+
+  test do
+    system bin/"drafter", "--help"
```
Comment (ilovezfs) [inline Formula/drafter.rb]: This should be a test that exercises, and verifies, the software's functionality once installed.

**Review by ilovezfs — CHANGES_REQUESTED** [review]

**Comment by ilovezfs** [comment]
@vassilevsky any update here?

**Comment by ilovezfs** [comment]
Thanks for the PR @vassilevsky! Please feel free to reopen a PR for this when it's using the `make install` target and has a test that exercises and verifies the functionality of the software.

**Comment by Kyslik** [comment]
Anything new on this?

https://github.com/apiaryio/drafter#install

**Pull request closed without merging**